packages/mdc-top-app-bar/constants.js

```diff
@@ -26,6 +26,8 @@ const strings = {
 
 /** @enum {string} */
 const cssClasses = {
+  FIXED_CLASS: 'mdc-top-app-bar--fixed',
+  FIXED_SCROLLED_CLASS: 'mdc-top-app-bar--fixed__scrolled',
```

Comment:
In BEM, `__` indicates a sub-element, but this appears to be a modifier class (for a modifier class - yo dawg).

We normally handle that by using a single `-` instead of `__`.

E.g.: `mdc-top-app-bar--fixed-scrolled`